lib/src/replay/element_parsers/element_data.dart

```diff
@@ -1,21 +1,30 @@
 import 'package:flutter/material.dart';
+import 'package:posthog_flutter/posthog_flutter.dart';
 
 class ElementData {
   List<ElementData>? children;
   Rect rect;
   String type;
+  Widget? widget;
 
   ElementData({
     this.children,
     required this.rect,
     required this.type,
+    this.widget,
```

Comment:
I'd just organize this that the required fields are first and then the optionals